src/main/java/org/apache/commons/math4/stat/descriptive/moment/Variance.java

```diff
@@ -224,6 +229,7 @@ public void clear() {
      * @return the variance of the values or Double.NaN if length = 0
      * @throws MathIllegalArgumentException if the array is null
      */
+    @SuppressWarnings("index:argument.type.incompatible") // values.length is @LTLengthOf(value = {"values"}, offset = {"0 - 1"})
```

Comment:
What's the actual error message here? I would expect that this and the other warnings like it should be handled by the Index Checker; if they are not, the fix should be relatively simple (do constant propagation?).